src/components/pages/data-wallet.tsx

```diff
@@ -26,7 +26,7 @@ export function DataWallet() {
       .then((res) => {
         const urlCreator = window.URL || window.webkitURL
         setQrCode(urlCreator.createObjectURL(res))
-        // TODO: When the workflow has been defined in the future we should revoke the object URL when done with:
+        //TODO: When the workflow has been defined in the future we should revoke the object URL when done with:
```

Comment:
@hugotiburtino should we keep this TODO or turn it into an issue / normal comment?

Reply:
please keep it as a normal comment